dispatcher/service/main.py

```diff
@@ -15,6 +26,53 @@
 logger = logging.getLogger(__name__)
 
 
+class MetricsNamespace:
+    def __init__(self, namespace):
+        self._namespace = namespace
+
+
+class MetricsServerSettings(MetricsNamespace):
+    def port(self):
+        return 8070
+
+
+class MetricsServer(MetricsServerSettings):
+    def __init__(self, namespace, registry):
+        MetricsNamespace.__init__(self, namespace)
+        self._registry = registry
+
+        self._server = None
+        self._tid = None
+
+    def start(self):
+        try:
+            # TODO: addr for ipv6 ?
+            self._server, self._tid = prometheus_client.start_http_server(self.port(), addr='localhost', registry=self._registry)
```

Comment:
For documentation, this starts a thread

https://github.com/prometheus/client_python/blob/e3bfa1f10195b6959c5f49503762d07a47e1654c/prometheus_client/exposition.py#L238